react-tutorial/src/App.js

```diff
@@ -1,25 +1,81 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
 
-function App() {
```

Comment:
The "App" component is quite standard on a React App.  I recommend you to create a new "components" folder and to move the Square and the Board to different files of that folder. Then, in App, you can use the Board component and everything should work.